lib/migration_data/active_record/migration.rb

```diff
@@ -15,6 +15,8 @@ def self.included(base)
         base.class_eval do
           def exec_migration_with_data(conn, direction)
             origin_exec_migration(conn, direction)
+            ::ActiveRecord::Base.connection.schema_cache.clear!
```

Comment:
But did you have a problem with this before? I'm not sure if it's needed - nobody complained about this before. So no problem - no code.